wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ParallelDeadlineGroup.java

```diff
@@ -38,11 +33,19 @@ public class ParallelDeadlineGroup extends Command {
    * @param otherCommands the other commands to be executed
    * @throws IllegalArgumentException if the deadline command is also in the otherCommands argument
    */
+  @SuppressWarnings("this-escape")
```

Comment:
If the method is static, I don't think there's a this-escape here.
If it's a linter false-positive, there should be a comment indicating that.